tests/conftest.py

```diff
@@ -312,6 +312,45 @@ def messages_successful_response() -> Dict[str, Any]:
     })
 
 
+@pytest.fixture(params=[
+        [{
+            'content': 'Hello!',
+            'rendered_content': '<p>Hello!</p>',
+            'timestamp': 1530129122,
+            'topic': 'hello world',
+            'user_id': 11
+        }],
+        [{
+            'content': 'Hello!',
+            'rendered_content': '<p>Hello!</p>',
+            'timestamp': 1530129122,
+            'topic': 'party at my houz',
+            'user_id': 5140
+        }, {
+            'content': 'Howdy!',
+            'content_html_diff': '<div><p><span class'
+                                 '="highlight_text_inserted">Howdy!</span>'
+                                 '</p> <p><span class='
+                                 '"highlight_text_deleted">Hello!</span>'
+                                 '</p></div>',
+            'prev_content': 'Hello!',
+            'prev_rendered_content': '<p>Hello!</p>',
+            'prev_topic': 'party at my houz',
+            'rendered_content': '<p>Howdy!</p>',
+            'timestamp': 1530129134,
+            'topic': 'party at my house',
+            'user_id': 5140
+        }],
+     ],
+     ids=['unedited_message', 'edited_message']
+)
+def message_history_fixture(request):
+    """
+    Return message history for a message.
+    """
+    return request.param
+
+
```

Comment:
The 'message history' is also needed in `TestEditHistoryView` which is why I went ahead with a fixture in `conftest.py`.